rollup/internal/controller/sender/sender.go

```diff
@@ -357,6 +366,10 @@ func (s *Sender) resubmitTransaction(tx *gethTypes.Transaction, baseFee, blobBas
 		originalGasPrice := tx.GasPrice()
 		gasPrice := new(big.Int).Mul(originalGasPrice, escalateMultipleNum)
 		gasPrice = new(big.Int).Div(gasPrice, escalateMultipleDen)
+		baseFeeInt := new(big.Int).SetUint64(baseFee)
+		if gasPrice.Cmp(baseFeeInt) < 0 {
+			gasPrice = baseFeeInt
```

Comment:
may as well add a multiplier here? the drawback would be over-charging by the multiplier.

Reply:
`gasPrice` is already multiplied. if after being multiplied it's still below `baseFee` we should bump it. I prefer not to multiplying again here.

Reply:
if the txn is submitted, and before it's included in a block, the block's base fee is updated by gas-oracle price update, the gas price will be < base fee again. same suspicion in another comment discussion.